libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1505,7 +1505,7 @@ bool AP_InertialSensor::gyro_calibrated_ok_all() const
 // Prearm check to verify that we have sane sched loop rates set based on Gyro backend rates
 bool AP_InertialSensor::pre_arm_check_gyro_backend_rate_hz(char* fail_msg, uint16_t fail_msg_len) const
 {
-#if AP_SCHEDULER_ENABLED
+#if AP_SCHEDULER_ENABLED && CONFIG_HAL_BOARD != HAL_BOARD_SITL
```

Comment:
can you try changing SIM_RATE_HZ? we should be able to get faster IMUs in SITL

Reply:
@tridge this is in realflight so my understanding is the gyro rate matches the loop rate?

Reply:
decided can do a return pass if AHRS_EKF_TYPE=10 in SITL